ReactNativeClient/lib/services/rest/Api.js

```diff
@@ -448,10 +448,11 @@ class Api {
 			output.parent_id = folder.id;
 		}
 
-		if ('source_url' in requestNote) output.source_url = requestNote.source_url;
-		if ('author' in requestNote) output.author = requestNote.author;
+		if (requestNote.source_url) output.source_url = requestNote.source_url;
+		if (requestNote.author) output.author = requestNote.author;
 		if ('user_updated_time' in requestNote) output.user_updated_time = Database.formatValue(Database.TYPE_INT, requestNote.user_updated_time);
 		if ('user_created_time' in requestNote) output.user_created_time = Database.formatValue(Database.TYPE_INT, requestNote.user_created_time);
+		if (requestNote.is_todo) output.is_todo = requestNote.is_todo;
```

Comment:
IMO it should rather be: ` 'is_todo' in requestNote`

Reply:
I am still new to the github workflow.  When I tried to merge there was a conflict with the addition of user-updated/created_time changes so I just added them in with mine.  Likewise the white space changes.